app/components/contributor-list/template.hbs

```diff
@@ -1 +1,7 @@
-{{contributorList}}
+{{contributor-list/contributor useLink=useLink contributor=first~}}{{~if rest.length ', ' ' '~}}
+{{#each rest as |contrib|}}
+    {{~contributor-list/contributor useLink=useLink contributor=contrib ~}}{{~if last ', ' ''~}}
+{{/each}}
+{{#if last}}
+    {{~t 'general.and'}} {{contributor-list/contributor useLink=useLink contributor=last}}
```

Comment:
This needs to updated to work with i18n.

Reply:
@binoculars I'm curious what the problem is. Is it just https://www.npmjs.com/package/i18n-list-generator or are there other potential problems? This is a surprisingly difficult question to google for, so if you have a page you could link, that would be awesome.

Reply:
@brianjgeiger That could be one viable solution. However, it would need to be integrated to work with Ember--perhaps as a helper or a component (maybe both?). I'm not aware of an out-of-the box solution for Ember, so we need to do our due diligence and come up with one.

Reply:
Oh, for sure. I just meant is the problem we're trying to solve that, or are there other intricacies with lists and foreign languages? I don't know the scope of the problem space and I've had a difficult time finding out if it's more than that.

Reply:
Inserting conjunctions (i.e. the word *and* in this case) may be tricky for non-latin based languages (some may require it, some may be optional, others may not have that concept) @laurenrevere and @cslzchen may have insight on that.

I'd consider it to be out of scope for this PR to translate/transliterate names.

From the broader perspective, I doubt we're ready for an RTL language like Arabic or Hebrew, but that is something we should address later on (but again would be out of scope here).

Since the component is being modified, now is the appropriate time to address the i18n list issue.

Reply:
Might be reasonable to adopt this approach: http://cldr.unicode.org/translation/lists